### PR TITLE
flagging duplicate log messages

### PR DESCRIPTION
**Pull request by ayeshaa475**

### CODE REVIEW

File: tests/flag_duplicate_log_messages_test.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import io
+
+from alogamous import analyzer, flag_duplicate_log_messages
+
+
+def test_flag_duplicate_log_messages():
```
Comment (laurenassour) [inline tests/flag_duplicate_log_messages_test.py:6]: Can we add a test for when there are no duplicate messages?

**Comment by benrady-aq** [comment]
Is this PR a duplicate of #49? How ironic!

**Comment by benrady-aq** [comment]
Maybe we should close this one?
